src/app/components/ATIAnalytics/index.test.tsx

```diff
@@ -1136,4 +1138,68 @@ describe('ATI Analytics Container', () => {
       });
     });
   });
+
+  describe('Reverb', () => {
+    it('should supply reverbParams when Reverb is enabled', () => {
+      const mockCanonical = jest.fn().mockReturnValue('canonical-return-value');
+      // @ts-expect-error - we need to mock these functions to ensure tests are deterministic
+      canonical.default = mockCanonical;
+
+      const {
+        metadata: { atiAnalytics },
+      } = articleDataNews;
+
+      // @ts-expect-error - only partial data required to manually set 'useReverb' to true
```

Comment:
Will these remain in the codebase for now or is there a timeline to adjust the code so that they're no longer needed?

Reply:
They will remain there as its only test mock data